pkg/volume/plugins.go

```diff
@@ -188,3 +196,24 @@ func EscapePluginName(in string) string {
 func UnescapePluginName(in string) string {
 	return strings.Replace(in, "~", "/", -1)
 }
+
+// FindPersistentPluginBySpec looks for a plugin that can support a given volume
+// specification.  If no plugins can support or more than one plugin can
+// support it, return error.
+func (pm *VolumePluginMgr) FindPersistentPluginBySpec(spec *api.Volume) (PersistentVolumePlugin, error) {
```

Comment:
I need the half the PV API to pass PersistentVolume ...

I will introduce this method in the PVManager PR, which is after API.